release_guidelines.md

```diff
@@ -56,7 +56,7 @@ An example release version `<VERSION>` could be `2.0.0`
 2. Build the dist files for release version
     - `npm run build`
 3. **[Cross Device]** Deploy dist files to the `<BASE_32_VERSION>` on AWS production:
-    - `aws s3 sync ./dist s3://onfido-assets-production/web-sdk/<BASE_32_VERSION>/ --exclude "*.html" --exclude "*.map" --acl public-read --delete`
+    - `aws s3 sync ./dist s3://tf-assets-20180717150854152100000001/<BASE_32_VERSION>/ --exclude "*.html" --acl public-read --delete`
```

Comment:
what is different from this one and 10.?

Reply:
The difference is that the new bundle generated by the build includes the release version instead of the release candidate version and it includes any changes done after the regression testing

Reply:
don't really follow :/ it seems to be a change and not an add, so the previous bundle should have needed this as well, no?